pkg/fission-cli/cmd/check/check.go

```diff
@@ -0,0 +1,37 @@
+package check
```

Comment:
Looks like we are missing copyright in this file. Can you please add it ?
Please do same for rest of new files.
Reference:

https://github.com/fission/fission/blob/3ac188124e348c2463a9e8b46bae9ab8967ce255/pkg/utils/gitrepo/gitrepo.go#L1